rmw_zenoh_cpp/src/detail/rmw_data_types.hpp

```diff
@@ -131,4 +129,78 @@ struct rmw_subscription_data_t
   std::condition_variable * condition{nullptr};
 };
 
+
+///==============================================================================
+
+// z_owned_closure_query_t
+void service_data_handler(const z_query_t * query, void * service_data);
+
+void client_data_handler(z_owned_reply_t * reply, void * client_data);
+
+
+struct saved_queryable_data
+{
+  explicit saved_queryable_data(z_owned_query_t query);
+  ~saved_queryable_data();
+
+  z_owned_query_t query;
+};
+
+///==============================================================================
+
+struct rmw_service_data_t
+{
+  unsigned int get_new_uid();
+
+  const char * zn_queryable_key;
```

Comment:
Nit: Follow naming style used in other places.
```suggestion
  const char * keyexpr;
```

Reply:
Done.